config/vufind/facets.ini

```diff
@@ -300,3 +300,8 @@ visual_facets = "callnumber-first,topic_facet"
 ; Use facet field names as keys and untranslated facet values as values.
 [HideFacetValue]
 ;format[] = "Book"
+
+; Prevent all but specific facet values from being displayed to the user.
+; Use facet field names as keys and untranslated facet values as values.
+[HideAllFacetValuesExcept]
```

Comment:
I'd rather name this ShowFacetValue or FacetValueFilter. Same with the parameter and variable names.

Reply:
Ok; I started with ShowFacetValue and renamed it afterwards because it's part of a "HideFacet" context and I felt it should have "hide" in it's name. Anyway I agree that ShowFacetValue sounds better and renamed it back.